backupServer.py

```diff
@@ -0,0 +1,128 @@
+import platform
+import os 
+import datetime
+import shutil
+
+
+date = datetime.datetime.now()
+system=  platform.system()
+
+
+
+BackupData= ''
+
+BackupVorm = input('hoe wilt u uw data backuppen opslaan 1) Local 2) Op de backupserver : ')
```

Comment:
De input die gegeven moet worden is niet goed duidelijk. Ik denk dat "hoe wilt u uw data backuppen opslaan Local (L) Op de backupserver (S): " beter zou zijn of "hoe wilt u uw data backuppen opslaan \n1) Local \n2) Op de backupserver \n: " Dit zou namelijk duidelijk maken dat ze los staan

Reply:
Kan, de style kan nog altijd aangepast worden,